tee-worker/app-libs/stf/src/trusted_call.rs

```diff
@@ -68,11 +68,13 @@ use sp_core::{
 	ed25519,
 };
 use sp_io::hashing::blake2_256;
-use sp_runtime::MultiAddress;
+use sp_runtime::{traits::ConstU32, BoundedVec, MultiAddress};
 use std::{format, prelude::v1::*, sync::Arc};
 
 pub type IMTCall = ita_sgx_runtime::IdentityManagementCall<Runtime>;
 pub type IMT = ita_sgx_runtime::pallet_imt::Pallet<Runtime>;
+pub type MaxAssertionLength = ConstU32<32>;
```

Comment:
This should not be greater than `4` , I don't think we even have `32` assertions.